contrib/metrics/supply_check/__init__.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import os
+import progressbar
+from slickrpc.rpc import Proxy
+
+from .deltas_mainnet import MainnetSupplyDeltas
+from .theoretical import Network, MAINNET
+
+COIN=100000000
+
+def TheoreticalAndEmpirical(zcashd, deltas, height, flag):
```

Comment:
Please document the meaning of `flag` by pointing to the docs for `getblock` and `getrawtransaction`.